src/hooks/PersistedQueryProvider.tsx

```diff
@@ -1,62 +0,0 @@
-import { createAsyncStoragePersister } from '@tanstack/query-async-storage-persister';
-import AsyncStorage from '@react-native-async-storage/async-storage';
-import { PersistQueryClientProvider } from '@tanstack/react-query-persist-client';
```

Comment:
@bruce-glazier In this PR or a follow-up, I think we can + should remove these two react-query persistence libraries. I think they'll be unused now.

Reply:
I'll grab it now. Sorry for the slop, extra sleepy today.